README.md

```diff
@@ -13,41 +13,29 @@ Windows      | Python 3.8      | [![TeamCity](https://dev.aegresco.com/teamcity/
 
 ---
 
-CSBuild is a language-agnostic build system focused on maximizing developer iteration time and providing tools for enabling
-developers to improve their build workflow. Currently, CSBuild is undergoing a complete rewrite to address some core architecture
-issues with the original iteration. It gets closer every day, but hasn't quite reached feature parity with the original CSBuild.
+CSBuild is a language-agnostic build system focused on maximizing developer iteration time and providing tools for enabling developers to improve their build workflow. Currently, CSBuild is undergoing a complete rewrite to address some core architecture issues with the original iteration. It gets closer every day, but hasn't quite reached feature parity with the original CSBuild.
 
 What it currently can do:
-- Build basic C++ files for Windows and Linux systems
-- Be extended with tools to work in any language (a new feature that old CSBuild did not support)
-- Support macro processing in all strings (i.e., SetOutputDirectory("{toolchainName}/{architectureName}/{targetName}"))
-- Generate project files for Visual Studio.
+- Build basic C++, java, objective-c, and assembly files for Windows, Mac, BSD, Linux, android, PS3, PS4, and PSVita systems
```

Comment:
This description is accurate... from a certain point of view.  We do support those languages and those platforms, but not all combinations of them.  It's fine for now if we don't want to nitpick, but we might want to improve the wording later on to avoid implying the wrong thing.

Reply:
updated